litestar/middleware/_internal/cors.py

```diff
@@ -50,7 +50,11 @@ async def __call__(self, scope: Scope, receive: Receive, send: Send) -> None:
             )
             await asgi_response(scope, receive, send)
         elif origin:
-            await self.app(scope, receive, self.send_wrapper(send=send, origin=origin, has_cookie="cookie" in headers))
+            await self.app(
+                scope,
+                receive,
+                self.send_wrapper(send=send, origin=origin, has_cookie="cookie" in headers),
+            )
```

Comment:
This change seems unrelated?

Reply:
Oh yeah that's probably just `ruff`, dunno if I had a weird config or something, I'll get rid

Reply:
You've added a trailing comma. Ruff then will always put the args on separate lines

Reply:
Oh, then I think what must have happened was my editor auto formatted with black on save which added the comma and then I ran ruff on top of that :D